app/src/main/java/com/vinh/dictionary_1/data/source/VEDictRepository.java

```diff
@@ -3,6 +3,7 @@
 import com.vinh.dictionary_1.data.model.Word;
 import com.vinh.dictionary_1.data.source.local.vedictdatabase.VEDictLocalDatasource;
 import io.reactivex.Flowable;
+import com.vinh.dictionary_1.MainApplication;
```

Comment:
ERROR: Unused import: com.vinh.dictionary_1.MainApplication. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports